xtask/src/cargo.rs

```diff
@@ -105,6 +105,11 @@ impl Package {
     ///
     /// If `update` is `true`, update the changelog for the release of the given version.
     pub fn changes(&self, update: bool) -> Result<String> {
+        if self.name == "ruma-macros" {
```

Comment:
Hmm, I thought we had a thing where ruma-macros automatically gets released together w/ ruma-common, and trying to release it independently results in an error?

Reply:
You remove it in https://github.com/ruma/ruma/pull/1689

Reply:
Aha! 😅